ReactNative/BrowserAPI/BrowserCliqz.swift

```diff
@@ -33,6 +33,11 @@ class BrowserCliqz: RCTEventEmitter, NativeModuleBase {
             return
         }
 
+        guard pref != AppConstants.PrefSendUsageData || Features.Telemetry.isEnabled else {
```

Comment:
It may be just me but regular `if pref == AppConstants.PrefSendUsageData && !Features.Telemetry.isEnabled` reads much better